_rules/SC4-1-2-aria-attr-value-valid.md

```diff
@@ -0,0 +1,252 @@
+---
+name: ARIA attributes have valid values
+test_type: atomic
+
+description: |
+  This rule checks that all ARIA 1.1 attributes have valid values
+
+success_criterion:
+- 4.1.2 # Name, role, value
+
+test_aspects:
+- DOM
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+Any HTML or SVG element that has one or more [ARIA 1.1 attribute](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) starting with `aria-`.
+
+### Expectation
+
+Each target element must have valid values for all ARIA 1.1 attributes. What values are allowed is defined per attribute in [ARIA 1.1, Definitions of States and Properties](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def). ARIA 1.1 attributes that have no value (`<p aria-expanded>`) or that are empty (`<p aria-expanded="">`) are allowed for all value types.
+
+**Note**: ID reference and ID reference list are required to have a valid value. This rule does not require that the element(s) exists.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
+- https://www.w3.org/TR/WCAG20-TECHS/ARIA5.html
+- https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value
+
+## Test Cases
```

Comment:
?? Remove repeated heading

Reply:
Also everywhere else we seem to have, instead of ## for Test Cases heading...

# Test cases

## Passed